transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/scheduling/persistence/SQLiteEventStore.java

```diff
@@ -63,18 +64,21 @@
   private final Clock wallClock;
   private final Clock monotonicClock;
   private final EventStoreConfig config;
+  private final Context context;
 
   @Inject
   SQLiteEventStore(
       @WallTime Clock wallClock,
       @Monotonic Clock clock,
       EventStoreConfig config,
-      SchemaManager schemaManager) {
+      SchemaManager schemaManager,
+      Context context) {
```

Comment:
Actually, taking a step back, looks like context is only needed to get packageName. Consider injecting the package name directly instead of injecting `Context`[1].

[1] https://github.com/google/guice/wiki/InjectOnlyDirectDependencies

Can we do something like the following?
```suggestion
      @PackageName String packageName) {
```

See https://github.com/google/guice/wiki/BindingAnnotations for details.